src/UserSettingsStore.js

```diff
@@ -171,22 +174,36 @@ export default {
         localStorage.setItem('mx_local_settings', JSON.stringify(settings));
     },
 
-    isFeatureEnabled: function(feature: string): boolean {
+    getFeatureById(feature: string) {
+        for (let i = 0; i < this.LABS_FEATURES.length; i++) {
+            const f = this.LABS_FEATURES[i];
+            if (f.id === feature) {
+                return f;
+            }
+        }
+        return null;
+    },
+
+    isFeatureEnabled: function(featureId: string): boolean {
         // Disable labs for guests.
         if (MatrixClientPeg.get().isGuest()) return false;
 
-        if (localStorage.getItem(`mx_labs_feature_${feature}`) === null) {
-            for (let i = 0; i < this.LABS_FEATURES.length; i++) {
-                const f = this.LABS_FEATURES[i];
-                if (f.id === feature) {
-                    return f.default;
-                }
-            }
+        const feature = this.getFeatureById(featureId);
+        if (!feature) {
+            console.warn('Unknown feature');
```

Comment:
please log what the feature is that's unknown. don't we normally use double-quotes for string constants, too?